services/docker-compose.yml

```diff
@@ -148,11 +148,36 @@ services:
         - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.healthcheck.timeout=1000ms
         - traefik.http.middlewares.${SWARM_STACK_NAME}_static_webserver_retry.retry.attempts=2
         - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.rule=hostregexp(`{host:.+}`)
-          && (PathPrefix(`/osparc`) || PathPrefix(`/s4l`) || PathPrefix(`/tis`)
-          || PathPrefix(`/transpiled`) || PathPrefix(`/resource`))
+          && (PathPrefix(`/osparc`,`/s4l`,`/tis`,`/transpiled`,`/resource`))
```

Comment:
THOUGHT: I wonder how could we guarantee sync of these in case of future addition or removal